packages/react-query-typechain/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": [
+    "@elementfi/eslint-config"
```

Comment:
Need this as a dependency in package.json as well, ie: `"@elementfi/eslint-config": "*"`